docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,245 @@
 <Subtitle>Changelogs for each Base UI release.</Subtitle>
 <Meta name="description" content="Changelogs for each Base UI release." />
 
+## v1.0.0-beta.5
+
+**Nov 13, 2025**
+
+### Accordion
+
+- **Breaking change:** Change `multiple` prop to be false by default and add a demo ([#3141](https://github.com/mui/base-ui/pull/3141))
+- Fix flaky exit transition ([#3101](https://github.com/mui/base-ui/pull/3101))
+
+### Alert Dialog
+
+- Fix `initialFocus` as function being called on close ([#2949](https://github.com/mui/base-ui/pull/2949))
+- Support detached triggers ([#2974](https://github.com/mui/base-ui/pull/2974))
+- Place `overflow: hidden` on `<body>` for overlay scrollbars ([#3083](https://github.com/mui/base-ui/pull/3083))
+
+### All Components
+
+- **Breaking change:** Replace `trackAnchor` with `disableAnchorTracking`.
+
+  If you were using `trackAnchor={false}`, be sure to update your usage to `disableAnchorTracking` instead.
+  ([#3188](https://github.com/mui/base-ui/pull/3188))
+
+- **Breaking change:** Rename `loop` to `loopFocus` ([#3186](https://github.com/mui/base-ui/pull/3186))
+- Fix type portability ([#2912](https://github.com/mui/base-ui/pull/2912))
+- Accepts `style` prop as function ([#3038](https://github.com/mui/base-ui/pull/3038))
+
+### Autocomplete
+
+- **Breaking change:** Refactor `alwaysSubmitOnEnter` to `submitOnItemClick` prop.
+
+  If you were using `alwaysSubmitOnEnter`, be sure to update your usage to `submitOnItemClick` instead.
+  ([#3018](https://github.com/mui/base-ui/pull/3018))
+
+- Prevent blocking filtering while composing text on Android ([#2944](https://github.com/mui/base-ui/pull/2944))
+- Add empty state to `List.State` ([#2934](https://github.com/mui/base-ui/pull/2934))
+- Fix `initialFocus` as function being called on close ([#2949](https://github.com/mui/base-ui/pull/2949))
+- Add `role="combobox"` to Trigger if Input is inside Popup ([#2973](https://github.com/mui/base-ui/pull/2973))
+- Fix stale `onItemHighlighted` data when filtering with `autoHighlight` ([#2829](https://github.com/mui/base-ui/pull/2829))
+- Add empty and side styling attributes on `Input` ([#2926](https://github.com/mui/base-ui/pull/2926))
+- Fix Value component return type for React 17 ([#3050](https://github.com/mui/base-ui/pull/3050))
```

Comment:
We can create a consistent format here for parts, it seems like we could use the angle brackets format (`<Autocomplete.Input>`, `<Autocomplete.Value>`, etc.)

Reply:
https://mui-org.slack.com/archives/C02EAR1R8CC/p1761229434828489?thread_ts=1761057279.401239&cid=C02EAR1R8CC

Reply:
The problem is that format doesn't work as nicely for parts and namespaces that we use. 

`DataGrid` is fine, but `DataGrid.Part` is not quite as clear

In Base UI, `Menu.Item.Props` / `Menu.Item.State` refer to types, not a component part. `<Menu.Item>` clearly refers to a component. Plus we also use this style: "Renders a `<div>` element." in the JSDoc, so that also matches.